config/config.go

```diff
@@ -24,6 +24,9 @@ type Configuration struct {
 	CacheClient HTTPClient `mapstructure:"http_client_cache"`
 	AdminPort   int        `mapstructure:"admin_port"`
 	EnableGzip  bool       `mapstructure:"enable_gzip"`
+	// GarbageCollectorLimit will serve as a fixed cost of memory that is going to be held garbage
+	// before a garbage collection cycle is triggered
+	GarbageCollectorLimit int `mapstructure:"garbage_collector_limit"`
```

Comment:
I think we can improve the wording for clarity, and a link to the Go issue might be a good idea. Perhaps something like:

// GarbageCollectorLimit allocates virtual memory (in bytes) which is not used by PBS but serves as a hack to trigger the garbage collector only when the heap reaches at least this size.

Reply:
Far better worded. Done